doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,7 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+* The ``CONFIG_KERNEL_DEBUG`` Kconfig option, which was used to enable ``printk()`` based debugging of the kernel internals, has been removed. The kernel now uses the standard Zephyr logging API at DBG log level for this purpose. The logging module used for the kernel is named ``os``.
```

Comment:
```suggestion
* The ``CONFIG_KERNEL_DEBUG`` Kconfig option, which was used to enable ``printk()`` based debugging of the kernel internals, has been removed. 
  The kernel now uses the standard Zephyr logging API at DBG log level for this purpose. 
  The logging module used for the kernel is named ``os``.
```
Please start every sentence on a new line.

Reply:
Also, please add a heading to be compatible with https://github.com/nrfconnect/sdk-nrf/pull/2639 - thus:
```suggestion
* Kernel:
 
  * The ``CONFIG_KERNEL_DEBUG`` Kconfig option ...
```